bin/cron/analyze_hoc_activity

```diff
@@ -140,11 +140,13 @@ def main
   petition_signatures = PEGASUS_REPORTING_DB_READONLY[:forms].where(kind: 'Petition').group_by(:hashed_email).count
 
   time = DateTime.now
+  # Pull current project_count to keep that current value.
   Properties.set :metrics, {
     created_at:               time,
     created_on:               time.to_date,
     petition_signatures:      petition_signatures,
     lines_of_code:            lines_of_code,
+    project_count:            Properties.get(:metrics)['project_count'],
```

Comment:
Just a comment for future readers that this will lead to `project_count` being set to `nil` when it was previously not recorded at all.  We have seen this in our test environments, for example, which has led to the `delete_if` added at https://github.com/code-dot-org/code-dot-org/pull/26807/files#diff-cfda98acedc510b51ec32e23bfad5d69R65